tst/testinstall/tilde.tst

```diff
@@ -82,4 +82,10 @@ true
 gap> bool := (rec1 < rec2);;
 gap> ForAll([1..10000], q -> ( (rec1 < rec2) = bool) );
 true
+gap> i := InputTextString( "local a; a := 10; return rec(a := a*10, b := ~);" );;
+gap> r := rec(a := ~, b := ReadAsFunction(i)(), c := ~.b.a);
+rec( a := ~, b := rec( a := 100, b := ~.b ), c := 100 )
+gap> name := Filename( DirectoriesLibrary("tst/testinstall"), "exampleReadAsFunction.txt" );;
+gap> r := rec(a := ~, b := ReadAsFunction(name)(), c := ~.b.a);
```

Comment:
Both calls to `ReadAsFunction` go through `READ_AS_FUNC` in the end, so I wonder what the point is in testing this twice? Sure, for full coverage we want to test both, but then perhaps in a .tst file that focuses on `ReadAsFunction` resp. on all functions from the Read family?

Reply:
Fine. Test removed.